### PR TITLE
drop extra cache stores

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: _includes/download-available-cache-stores-band.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="component-slim">
   <div class="grid-wrapper">
     <div class="width-12-12">
-      <h4>Available Cache Stores</h4>
-      <p>The following cache stores are part of the core Infinispan distribution and therefore are developed and released in-sync with Infinispan itself. Most of these cache stores are included in the <code>infinispan-embedded</code> JAR. Those which require additional dependencies are available in separate artifacts.</p>
+      <h4>Infinispan Cache Stores</h4>
+      <p>Infinispan offers several cache store implementations that are available with each release. Most cache stores are included in the <code>infinispan-embedded</code> JAR. Some others require additional dependencies and are available in separate artifacts.</p>
```
Comment (pruivo) [inline _includes/download-available-cache-stores-band.html:5]: `<code>infinispan-embedded</code> JAR` has been removed a while back

Comment (oraNod) [reply _includes/download-available-cache-stores-band.html]: nice one! I was going to ping you to ask so well spotted so quickly.

Comment (oraNod) [reply _includes/download-available-cache-stores-band.html]: hopefully the rewording sounds alright.

**Review by pruivo — APPROVED** [review]

**Review by wburns — APPROVED** [review]
Looks good, just confirming safe to integrate.

**Comment by oraNod** [comment]
@wburns Yes, I built locally and we're good to go. I'll go ahead and merge while I'm here. Thanks for the review.

**Pull request merged**